numbat/src/ffi/datetime.rs

```diff
@@ -30,15 +29,16 @@ pub fn format_datetime(mut args: Args) -> Result<Value> {
     let dt = datetime_arg!(args);
 
     let mut output = String::new();
-    write!(output, "{}", dt.format(&format)).map_err(|_| RuntimeError::DateFormattingError)?;
+    write!(output, "{}", dt.strftime(&format)).map_err(|_| RuntimeError::DateFormattingError)?;
 
     return_string!(output)
 }
 
 pub fn get_local_timezone(_args: Args) -> Result<Value> {
-    let local_tz = datetime::get_local_timezone_or_utc().to_string();
+    let local_tz = datetime::get_local_timezone_or_utc();
+    let tz_name = local_tz.iana_name().unwrap_or("<unknown timezone>"); // TODO
```

Comment:
To do: can `Timezone::system().iana_name()` fail? Probably doesn't hurt to leave the `unwrap_or` in place.

Reply:
Yes, it definitely can. A specific example is when `/etc/localtime` is a valid TZif file but _not_ a symlink. Since a TZif file itself does not contain the actual time zone identifier, there is no feasible way for Jiff to return the name of the time zone _but_ it can do still DST safe arithmetic with it.